### ISSUE TITLE
node 19.6.1:Uncaught SyntaxError: Unexpected identifier 'SchemaOptions'

### ISSUE THREAD

**Issue by mathiasrichter** [siqueirarenan/shacl-jsonschema-converter#1]
Very excited to try your work! Unfortunately, I'm running into issues with  node v19.6.1:

Did `npm i shacl-jsonschema-converter` and now trying to do

```
const { getSchemasFromTtl } = require('shacl-jsonschema-converter');
```

but getting 

```
node_modules/shacl-jsonschema-converter/converter.ts:1
type SchemaOptions = {
     ^^^^^^^^^^^^^

Uncaught SyntaxError: Unexpected identifier 'SchemaOptions'

```

Can you advise please?

**Comment by siqueirarenan** [comment]
Hi, thanks for the feedback!

there was an issue with the packaging, fixed in [version 0.1.3](https://www.npmjs.com/package/shacl-jsonschema-converter).

**Comment by mathiasrichter** [comment]
Cool, many thanks for the quick turnaround! Can’t wait to try it!
